### PR TITLE
Add info about minimum deposit value

### PR DESCRIPTION
**Pull request by michalsmiarowski**
Closes: #390 

The dust threshold enforced by the minting contracts isn't displayed in the app, allowing users to mistakenly send ~$230 at today's prices, leaving it stuck for up to 9 months.

We are adding info about minimum deposit value (0.01 BTC) in step 2 of the minting flow and in minting timeline.

I've also added a non-breaking space (`&nbsp;`) between the value and `BTC` (`0.01&nbsp;BTC`) so that those two words alway stay in the same line.

Todo: 
- [ ] Make the error when doing a reveal more readable if too little BTC were sent. Also let the user know about the refund instructions.

### CODE REVIEW

File: src/pages/tBTC/Bridge/MintingCard/MakeDeposit.tsx [hunk 1]
```diff
@@ -53,8 +53,8 @@ const MakeDepositComponent: FC<{
         subTitle="Make your BTC deposit"
       />
       <BodyMd color="gray.500" mb={6}>
-        Use this generated address to send any amount of BTC you want to mint as
-        tBTC.
+        Use this generated address to send any amount larger than 0.01&nbsp;BTC,
```
Comment (r-czajkowski) [inline src/pages/tBTC/Bridge/MintingCard/MakeDeposit.tsx:56]: The amount may be larger than or equal to deposit dust threshold.

https://github.com/keep-network/tbtc-v2/blob/a511c31bcd44c44d4fc28dbe07f98b1648954da0/solidity/contracts/bridge/Deposit.sol#L248-L251

File: src/pages/tBTC/Bridge/MintingCard/MintingTimeline.tsx [hunk 2]
```diff
@@ -55,7 +55,7 @@ export const MintingTimelineStep2: FC<MintingTimelineStepProps> = ({
       stepText="Step 2"
       helperLabelText="ACTION ON BITCOIN NETWORK"
       title="Make a BTC deposit"
-      description="Send any amount of BTC to this unique BTC Deposit Address. The amount you send is the amount will be minted as tBTC."
+      description="Send any amount larger than 0.01&nbsp;BTC to this unique BTC Deposit Address. The amount you send will be minted as tBTC."
```
Comment (r-czajkowski) [inline src/pages/tBTC/Bridge/MintingCard/MintingTimeline.tsx:58]: Same as above.

File: src/pages/tBTC/Bridge/MintingCard/MakeDeposit.tsx (hunk 1 above)
Comment (r-czajkowski) [inline src/pages/tBTC/Bridge/MintingCard/MakeDeposit.tsx:56]: The `deposit dust threshold` is a governable param so we should fetch this value from the contract. Ofc for release we are good to go here.

Comment (michalsmiarowski) [reply src/pages/tBTC/Bridge/MintingCard/MakeDeposit.tsx]: c6794cd989efa030d3f9fb2185d04737a660e388

File: src/pages/tBTC/Bridge/MintingCard/MintingTimeline.tsx (hunk 2 above)
Comment (michalsmiarowski) [reply src/pages/tBTC/Bridge/MintingCard/MintingTimeline.tsx]: c6794cd989efa030d3f9fb2185d04737a660e388

File: src/pages/tBTC/Bridge/MintingCard/MakeDeposit.tsx (hunk 1 above)
Comment (michalsmiarowski) [reply src/pages/tBTC/Bridge/MintingCard/MakeDeposit.tsx]: Let's address it in a separate PR after the release

**Review by r-czajkowski — APPROVED** [review]

**Pull request merged**